shared/src/data/default-state/material-templates.ts

```diff
@@ -0,0 +1,36 @@
+import { CanCaterFor } from '../../models/utils';
+import { MaterialTemplate } from '../../models/material-template';
+import type { MaterialType } from '../../models/utils/material-type';
+
+const defaultMaterialTemplate = MaterialTemplate.create(
+    'default',
+    {
+        url: '/assets/material.svg',
+        height: 40,
+        aspectRatio: 1,
+    },
+    CanCaterFor.create(2, 0, 0, 'and'),
+    2.5,
+    5.5,
+    false
+);
+
+const bigMaterialTemplate = MaterialTemplate.create(
+    'big',
+    {
+        url: '/assets/material.svg',
+        height: 80,
+        aspectRatio: 1,
+    },
+    CanCaterFor.create(2, 2, 0, 'and'),
+    15,
+    0,
+    true
+);
+
+export const materialTemplateMap: {
+    [key in MaterialType]: MaterialTemplate;
+} = {
+    default: defaultMaterialTemplate,
+    big: bigMaterialTemplate,
```

Comment:
@christianzoellner Do you think we should add a "big" material to the GW-SAN?
* The `default material` can treat _two patients <=red_ (_2 red_ or _1 red +1 yellow_ or _1 red + 1 green_, _2 yellow_ or ... ) in a range of up to 5.5 (We don't have a standardized unit for this yet ^^).
* The `big material` can treat _two patients that are <=red and two patients that are <= yellow_ (_2 red and 2 yellow_ or _4 yellow_ or _1 red and 1 green_ or ...) in a bigger range. 

Currently, the GW-SAN has 8 `default materials`.

Reply:
"big" materials that sounds like a cool idea, your poposed stats are also ok for a start.

Reply:
As you might have notices currently the material is "underutilized" in the analog version of FüSim, since quantitatively it is rarely a bottleneck, and geometry is not really considered.

For the future, however, I can imagine that especially in cases were our digital FüSim a) leads to players considering physical map positions and b) enforces patient transport durations on the local map, the material placement could become an issue for situations where patients are spread out over an area. 
Especially for such scenarios, having big materials sound really useful, since it forces you even more to organize and transport your patients.

So, it it's not a lot of trouble, let's keep the `big material` as a prototype. It will do now harm and possibly allow some finer game mechanics in the future, as mentioned above. And it is also more comfortable for players to have to move around less materials when unloading a GW San.

Reply:
@Dassderdie @christianzoellner I would propose to give the GW San 4 big material, about the range, I would put it above the normal material, as personnel would probably take something out and take it to a patient around them. Could be smaller than the 15 (that a notarzt gets) but bigger than the 5.5.

I would not change the picture, even if it does look awkward/...

Changing images would be another PR or maybe later a graphic redesign could happen.
If somebody has a better picture, that has a license similiar to the other pictures, we could use that:
@christianzoellner what you described was something like this [https://media.schaefer-shop.de/is/image/schaefershop/shop500/transportbox-alutec-classic-30-aluminium-30-l-l-430-x-b-335-x-h-270-mm-zylinderschloesser-img_SI_138354_A_cut](https://media.schaefer-shop.de/is/image/schaefershop/shop500/transportbox-alutec-classic-30-aluminium-30-l-l-430-x-b-335-x-h-270-mm-zylinderschloesser-img_SI_138354_A_cut) right?

Reply:
@anonym-HPI The picture below is more accurate with regards to the overall size and the size ratio between the overall boy and the handles. Hoewver, the pins at the top and on the handles should be orange instead of blue!
https://certeo.scene7.com/is/image/certeo/000017665924_PRD_org_all?$s7product$

Reply:
> @Dassderdie @christianzoellner I would propose to give the GW San 4 big material, about the range, I would put it above the normal material, as personnel would probably take something out and take it to a patient around them. Could be smaller than the 15 (that a notarzt gets) but bigger than the 5.5.

@anonym-HPI The values sound reasonable. With regard to range, I thing it should be a bit lower than a Notarzt, so maybe something in the range from 10 to 12?

Reply:
Ok, then lets use it that way. @christianzoellner are you okay with merging it with just the old image bigger and another image is put in later (maybe in the next bachelor project, maybe also creating a new helicopter image)?

Reply:
@anonym-HPI yep, I am ok with the intermediate picture.

please create an issue for skipped tasks such as the image creation.

Reply:
![image](https://user-images.githubusercontent.com/18506183/189901293-b70b76cf-d39c-4a65-aa72-6121343a4722.png)

These are now the maximum ranges of personnel and materials:
* notarzt: 15
* standard material and (most) other personnel: 5.5
* big material: 10

Reply:
> please create an issue for skipped tasks such as the image creation.

#521